src/Lumi/Components2/QRCode.purs

```diff
@@ -0,0 +1,67 @@
+module Lumi.Components2.QRCode where
+
+import Prelude
+import Data.Newtype (class Newtype)
+import Data.Nullable as Nullable
+import Effect (Effect)
+import Lumi.Styles (toCSS)
+import Lumi.Styles.QRCode as Styles.QRCode
+import Lumi.Styles.Theme (LumiTheme, useTheme)
+import React.Basic.DOM as R
+import React.Basic.Emotion as E
+import React.Basic.Hooks (Hook, JSX, ReactComponent, Ref, UseContext, UseRef, coerceHook, element, useRef)
+import React.Basic.Hooks as React
+import Web.DOM (Node)
+
+newtype UseQRCode hooks
+  = UseQRCode (UseRef (Nullable.Nullable Node) (UseContext LumiTheme hooks))
+
+derive instance ntUseQRCode :: Newtype (UseQRCode hooks) _
+
+data ErrorCorrectLevel
+  = L
+  | M
+  | Q
+  | H
```

Comment:
Can we use more descriptive names for these constructors?

Reply:
The levels are from the QR code lib which come from their definition in the spec:

L[ow] - up to 7% damage
M[edium] - up to 15% damage
Q[uality] - up to 25% damage
H[igh] - up to 30% damage

Would you prefer the expanded names (Medium) or something more specific about the damage threshold (ECL15 or something)?

Reply:
Ah, I see, so the single letter names _are_ the actual standard it seems (https://en.wikipedia.org/wiki/QR_code#Error_correction). It's fine by me to keep it as is then.